configuration/configuration.go

```diff
@@ -116,8 +118,20 @@ func getConfigurationPath(cliArguments cliparser.CliArguments, logger *logger.Lo
 		notifyUserAboutConfigurationFile(path, logger)
 		return path, nil
 	} else {
-		return "", errors.New("Configuration file could not be read!")
+		answer:=""
+		fmt.Println("File not found. Make new configuration file? Y/N")
+		fmt.Scan(&answer)
+		if answer=="Y" {
+			configurevalidator.ShowRegions()
```

Comment:
You can replace all this configurevalidator functions with single function inside configurevalidator package. Name it CreateConfiguration or Configure and do not expose so many functions outside package.